Classes/Validator/ErrorCheck/FileMaxCount.php

```diff
@@ -73,7 +73,7 @@ public function check(): string {
         if (!is_array($info['name'][$this->formFieldName])) {
           $info['name'][$this->formFieldName] = [$info['name'][$this->formFieldName]];
         }
-        if (strlen($info['name'][$this->formFieldName][0]) > 0 && count((array) $info['name'][$this->formFieldName]) + count((array) ($files[$this->formFieldName] ?? [])) > $maxCount) {
+        if (strlen((string)$info['name'][$this->formFieldName][0]) > 0 && count((array) $info['name'][$this->formFieldName]) + count((array) ($files[$this->formFieldName] ?? [])) > $maxCount) {
```

Comment:
Also, I would prefer you to try this.

`
        if (strlen($info['name'][$this->formFieldName][0] ?? '') > 0 && count((array) $info['name'][$this->formFieldName]) + count((array) ($files[$this->formFieldName] ?? [])) > $maxCount) {
`

Should do the same in a null safe way.